terraform/environment/dns.tf

```diff
@@ -0,0 +1,29 @@
+variable "subdomains" {
+  type = list(string)
+  default = [
+    "nginz-https",
+    "nginz-ssl",
+    "webapp",
+    "assets",
+    "account",
+    "teams"
+  ]
```

Comment:
suggestion: defaulting to this list is still implicit from the env config's perspective. I think en empty list makes sense here.

Reply:
Sure.